toonz/sources/toonz/main.cpp

```diff
@@ -109,6 +109,10 @@ const char *applicationFullName = "OpenToonz 1.1.3";
 const char *rootVarName         = "TOONZROOT";
 const char *systemVarPrefix     = "TOONZ";
 
+QString workingDirectoryTmp  = QDir::currentPath();
+QByteArray ba                = workingDirectoryTmp.toLatin1();
+const char *workingDirectory = ba.data();
```

Comment:
Maybe obtaining the path to the working directory can be moved from here to the constructor of `EnvGlobals` as well ?